src/services/dashboards/widgets/widget-helper.ts

```diff
@@ -12,7 +12,7 @@ import type { HistoryDataModel, XYChartData } from '@/services/dashboards/widget
 import { BASE_WIDGET_CONFIGS, CONSOLE_WIDGET_CONFIGS } from '@/services/dashboards/widgets/widget-config-list';
 
 const mergeCustomizer = (val1, val2) => {
-    if (Array.isArray(val1)) return val1.concat(val2);
+    if (Array.isArray(val1)) return [...new Set(val1.concat(val2))];
```

Comment:
description: prevent duplicated items. for example-
```
widget_options_schema: {
  default_properties: ['group_by', 'group_by'],
}
```

Reply:
Question: I wonder why you used Set Object together instead of just using the spread operator!

Reply:
because Set removes duplicated items :)